### PR TITLE
feat(postgresq-osmdbt): created

### PR DESCRIPTION
**Pull request by CptSchnitz**

### CODE REVIEW

**Comment by asafMasa** [comment]
Should this image be in "shared-images" repo?

**Comment by syncush** [comment]
> Should this image be in "shared-images" repo?

Yes.

File: postgresql-osmdbt/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM bitnami/postgresql:14.6.0 as build
+USER root
+WORKDIR /app
+RUN apt update -y && apt install -y git build-essential cmake
+RUN git clone https://github.com/openstreetmap/osmdbt.git &&  \
```
Comment (syncush) [inline postgresql-osmdbt/Dockerfile:5]: I think we need to checkout here with a specific version

File: postgresql-osmdbt/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# postgresql-osmdbt
+This container is a modification of the [bitnami/postgresql container](https://github.com/bitnami/containers/tree/main/bitnami/postgresql) so it contains the [osmdbt postgresql plugin](https://github.com/openstreetmap/osmdbt) for the replication slot.
+
+In order to deploy the image on Openshift you should use the [bitnami/postgresql](https://github.com/bitnami/charts/tree/main/bitnami/postgresql) helm chart with the modifications present in the `values.yaml` file.
```
Comment (syncush) [inline postgresql-osmdbt/README.md]: let's add a section that if a programmer wants to change value or add a value, he should look up the helm chart first of bitnami before adding it here.

**Review by syncush — CHANGES_REQUESTED** [review]

File: postgresql-osmdbt/Dockerfile (hunk 1 above)
Comment (CptSchnitz) [reply postgresql-osmdbt/Dockerfile:5]: done

File: postgresql-osmdbt/README.md (hunk 2 above)
Comment (CptSchnitz) [reply postgresql-osmdbt/README.md]: done

**Review by syncush — APPROVED** [review]

**Pull request merged**